deploy/operator.yaml

```diff
@@ -14,6 +14,17 @@ spec:
         name: managed-velero-operator
     spec:
       serviceAccountName: managed-velero-operator
+      affinity:
```

Comment:
This change needs to be made here as well:  https://github.com/openshift/managed-cluster-config/blob/master/deploy/managed-velero-operator/135-velero.Deployment.yaml

At least until https://github.com/openshift/managed-cluster-config/pull/171 is merged.

Reply:
Thanks for pointing this out. Created [PR for managed-cluster-config](https://github.com/openshift/managed-cluster-config/pull/177).